integration_tests/dbt_project.yml

```diff
@@ -45,63 +45,63 @@ vars:
 seeds:
   quote_columns: false
   snowplow_web_integration_tests:
-    +schema: "snowplow_int_tests"
+    +schema: "snplw_web_int_tests"
     source:
       snowplow_web_events:
         +column_types:
-          app_id: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
+          app_id: "{{ 'string' if target.type == 'bigquery' else 'varchar' }}"
           etl_tstamp: timestamp
           collector_tstamp: timestamp
           dvce_created_tstamp: timestamp
-          user_fingerprint: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          ip_isp: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          ip_organization: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          ip_domain: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          ip_netspeed: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          page_urlfragment: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          refr_urlfragment: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          refr_medium: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          refr_source: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          refr_term: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          mkt_term: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          mkt_content: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          se_label: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
-          se_property: "{{ 'varchar' if target.type == 'redshift' else 'string' }}" 
+          user_fingerprint: "{{ 'string' if target.type == 'bigquery' else 'varchar' }}"
```

Comment:
Does this change not change snowflake from `string` in 0.3.1 to `varchar` in 0.4.0? Is that ok?

Reply:
Yeh you are correct it does change it but thankfully for Snowflake `string` and `varchar` are [synonymous](https://docs.snowflake.com/en/sql-reference/data-types-text.html#string-text). It is only BigQuery that requires `string` datatype.